### ISSUE TITLE
[Typography] Add responsive type

### ISSUE THREAD

**Issue by ramtinabadi** [Vieolo/vieolo-ui#213]
The regular typography classes are applied universally.
There should be props enabling the implementer to define different types for different screen sizes (xl, la, md, sm).
The CSS classes should have variations that are applied only on the media queries.
The props for different screen sizes can be passed individually or the `type` prop can accept both a string (which is used for universal type) or an object where `xl` is mandatory but other sizes can be defined optionally.
